gpeople/tests/main_test.bal

```diff
@@ -291,25 +279,28 @@ function testUpdateContactGroup() {
     }
 }
 
-@test:Config {dependsOn: [testUpdateContactGroup]}
-function testDeleteContactGroup() {
-    log:printInfo("Running Delete Contact Group Test");
-    var deleteContactGroup = googleContactClient->deleteContactGroup(contactGroupResourceName);
-    if (deleteContactGroup is ()) {
-        test:assertTrue(true, msg = "Delete Contact Group Failed");
-    } else {
-        test:assertFail(msg = deleteContactGroup.message());
-    }
-}
-
 @test:AfterSuite { }
 function afterSuite() {
     log:printInfo("AfterSuite");
     runtime:sleep(10);
     var deleteContact = googleContactClient->deleteContact(beforeSuiteResourceName);
     if (deleteContact is ()) {
-        test:assertTrue(true, msg = "Delete Contact Failed");
+        log:printInfo("Contact deleted");
+    } else {
+        log:printInfo("Delete Contact Failed");
+    }
+    runtime:sleep(10);
+    deleteContact = googleContactClient->deleteContact(contactResourceName);
+    if (deleteContact is ()) {
```

Comment:
What about checking error instead of (). It will be more readable. 

if (deleteContact is error) {
   log:printInfo("Delete Contact Failed");
}

Also we should try to be informative. Can we print the error here as well?

Reply:
That is more readable